code/modules/modular_computers/file_system/programs/generic/ntsl2_interpreter.dm

```diff
@@ -3,6 +3,7 @@
 	filedesc = "NTSL2+ Interpreter"
 	extended_desc = "This program is used to run NTSL2+ programs."
 	program_icon_state = "generic"
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET | PROGRAM_STATIONBOUND | PROGRAM_WRISTBOUND
```

Comment:
Undocumented change.
```suggestion
	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET
```
If you want to use NTSL Programs, you should imho get a more capable device than a wristbound.

Reply:
This was introduced in a previous PR.